swift/Sources/SignalClient/zkgroup/ByteArray.swift

```diff
@@ -0,0 +1,55 @@
+//
+// Copyright 2019-2021 Signal Messenger, LLC.
+// SPDX-License-Identifier: AGPL-3.0-only
+//
+
+import Foundation
+import SignalFfi
+
+public class ByteArray {
+    private let contents: [UInt8]
+
+    init(newContents: [UInt8], expectedLength: Int, unrecoverable: Bool = false) throws {
```

Comment:
What is the purpose of "unrecoverable" here?  It looks like it's not used.

Reply:
Don't worry, it's going away in one of the follow-ups.